Okta.Xamarin/Okta.Xamarin/OidcClient.cs

```diff
@@ -337,9 +339,15 @@ protected virtual string GetBasePath()
             return $"{Config?.OktaDomain}/oauth2/v1";
         }
 
-        protected virtual string GetAuthorizationServerBasePath(string authorizationServerId = "default")
+        protected virtual string GetAuthorizationServerBasePath(string authorizationServerId = null)
         {
-            return $"{Config?.OktaDomain}/oauth2/{authorizationServerId}/v1";
+            string domain = Config?.OktaDomain;
+            if (domain.EndsWith("/"))
```

Comment:
Can `domain` be null here? Say if `Config` is not set at the line 344

Reply:
Good point!  I'm pretty sure the config will actually throw on instantiation if domain is null, but it makes sense to protect against it here also.